ROADMAP.md

```diff
@@ -1,62 +1,191 @@
-    - We generate artifacts for old versions during merges to the libp2p repositories https://github.com/libp2p/test-plans/issues/35#issuecomment-1254991985
\ No newline at end of file
+# test-plans roadmap Q4’22/Q1’23 <!-- omit in toc -->
+
+```
+Date: 2022-10-18
+Status: In Progress
+Notes: This document is still in review will may be heavily modified based on stakeholder feedback. Please add any feedback or questions in:
+<Tracking Issue - perhaps this is not necessary for this round>
+```
+
+## Table of Contents <!-- omit in toc -->
+
+- [About the Roadmap](#about-the-roadmap)
+  - [Vision](#vision)
+  - [Sections](#sections)
+- [🛣️ Milestones](#️-milestones)
+  - [2022](#2022)
+    - [Early Q4 (October)](#early-q4-october)
+    - [Mid Q4 (November)](#mid-q4-november)
+    - [End of Q4 (December)](#end-of-q4-december)
+  - [2023](#2023)
+    - [Early Q1 (January)](#early-q1-january)
+    - [Mid Q1 (February)](#mid-q1-february)
+    - [End of Q1 (March)](#end-of-q1-march)
+  - [Up Next](#up-next)
+- [📖 Appendix](#-appendix)
+  - [A. Multi-dimensional Testing/Interop Visibility](#a-multi-dimensional-testinginterop-visibility)
+    - [1. User configured interop tests & dashboard](#1-user-configured-interop-tests--dashboard)
+    - [2. Interop test plans for all existing/developing features](#2-interop-test-plans-for-all-existingdeveloping-features)
+    - [3. Rock solid libp2p releases](#3-rock-solid-libp2p-releases)
+    - [4. Canonical interop tests & dashboard](#4-canonical-interop-tests--dashboard)
+  - [B. Hardening test infrastructure](#b-hardening-test-infrastructure)
+    - [1. Track test suite stability](#1-track-test-suite-stability)
+    - [2. Design process for adding new tests](#2-design-process-for-adding-new-tests)
+    - [3. Be the home for all interop tests](#3-be-the-home-for-all-interop-tests)
+    - [4. Stable build process](#4-stable-build-process)
+  - [C. Future-proof Benchmarking](#c-future-proof-benchmarking)
+    - [1. Benchmarking using nix-builders](#1-benchmarking-using-nix-builders)
+    - [2. Benchmarking using remote runners](#2-benchmarking-using-remote-runners)
+  - [D. Expansive protocol test coverage](#d-expansive-protocol-test-coverage)
+    - [1. AutoNat](#1-autonat)
+    - [2. Hole Punching](#2-hole-punching)
+    - [3. Custom topologies](#3-custom-topologies)
+    - [4. MTU Fixes](#4-mtu-fixes)
+
+## About the Roadmap
+
+### Vision
+The tenets of the libp2p team are:
+- Security
+  - We strive to ensure that libp2p is reliable & secure for users & applications. We seek to minimize vulnerabilities and act quickly to address issues.
+- Stability
+  - We do not break deployed features and maintain cross version compatibility across libp2p releases.
+- Performance
+  - We aim to make libp2p as performant as possible and strive to ensure comparable performance to widely used protocols on the Internet. We avoid regressions and performance degradations in new releases.
```

Comment:
These seem like they belong in the specs ROADMAP and we make that the canonical place for them?

I think another tenet for the project is "specs" first.